core/chains/evm/config/config.go

```diff
@@ -90,16 +90,23 @@ type ChainScopedOnlyConfig interface {
 
 //go:generate mockery --name ChainScopedConfig --output ./mocks/ --case=underscore
 type ChainScopedConfig interface {
-	config.GeneralConfig
+	config.BasicConfig
```

Comment:
A better name :+1:

Reply:
I don't recall if I explained the nuance of this change anywhere, but for posterity: I moved some methods between the two interfaces to align the abstractions with how we were using them. The old sets were intuitive for humans, but this way some uses could be reduced to just one interface or the other. (and we avoid another set of unimplemented/panicing v2 methods)